### ISSUE TITLE
center  Centernet loss

### ISSUE THREAD

**Issue by wangx1996** [tianweiy/CenterPoint#178]
Hi, I just read the code about the loss part, I find it is a little bit different to the original centernet loss or cornernet loss.

`    mask = mask.float()
    gt = torch.pow(1 - target, 4)
    neg_loss = torch.log(1 - out) * torch.pow(out, 2) * gt
    neg_loss = neg_loss.sum()
`

cornernet :      neg_loss = torch.log(1 - neg_pred) * torch.pow(neg_pred, 2) * neg_weights
centernet:       neg_loss = torch.log(1 - pred) * torch.pow(pred, 2) * neg_weights * neg_inds

Is there missing one parameter?

**Comment by tianweiy** [comment]
I don't think so. What is missing? neg_weight is just torch.pow(1 - target, 4)
